src/pages/gd/Swap/SwapCelo/UniSwap.tsx

```diff
@@ -113,6 +126,8 @@ export const UniSwap = (): JSX.Element => {
                     const { tokenAddress } = info
                     const symbol = tokenSymbols[tokenAddress]
                     const summary = symbol ? `Approved spending of ${symbol}` : 'Approved spending'
+                    const type = symbol ? 'sell' : 'buy'
+                    sendData({ event: 'swap', action: 'swap_approve', type, network })
```

Comment:
why do we need `network` in the events?

Reply:
because swap_approve is the general swap action that is being send on all networks
so the distinction is relevant